appeals/web/src/server/appeals/appeal-details/linked-appeals/add/add.mapper.js

```diff
@@ -124,7 +124,7 @@ export function addLinkedAppealCheckAndConfirmPage(appealData, sessionData) {
 								items: [
 									{
 										text: 'Change',
-										href: `/appeals-service/appeal-details/${appealData.appealId}/linked-appeals/add/lead-appeal`
+										href: `/appeals-service/appeal-details/${appealData.appealId}/linked-appeals/add/lead-appeal?backUrl=/appeals-service/appeal-details/${appealData.appealId}/linked-appeals/add/check-and-confirm`
```

Comment:
As above, ideally the new utility should be used to add the back link query: https://github.com/Planning-Inspectorate/appeals-back-office/blob/main/appeals/web/src/server/lib/url-utilities.js#L50